src/main/scala/diplomaticobjectmodel/model/OMRTLModule.scala

```diff
@@ -31,22 +35,79 @@ trait Asynchronous extends Synchronicity
 case class OMRTLReset(
   activeEdge: Option[OMSignalAssertionLevel],
   clock: String, // This will always be the name of the clock signal on the to p-level module
-  synchronicity: Option[Synchronicity]
-)
+  synchronicity: Option[Synchronicity],
+  _types: Seq[String] = Seq("OMClockReset", "OMRTLComponent", "OMCompoundType")
+) extends OMRTLComponent
 
+case class OMReset(
+  description: Option[String],
+  name: String,
+  activeEdge: Option[OMSignalAssertionLevel],
+  clockName: String, // This will always be the name of the clock signal on the top-level module
+  synchronicity: Option[Synchronicity],
+  uid: Int,
+  _types: Seq[String] = Seq("OMReset", "OMSignal", "OMRTLComponent", "OMCompoundType")
+) extends OMSignal
+
+// Status
+trait OMStatus extends OMSignal {
+  def description: Option[String]
+  def name: String
+  uid: Int
+}
+
+trait OMCoreStatus extends OMStatus {
+  def description: Option[String]
+  def name: String
```

Comment:
These two fields are already defined in `OMStatus`, so you don't need to repeat them here.